recipes/libpyvinyl/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set version = "1.3.0" %}
+
+package:
+  name: libpyvinyl
+  version: {{ version }}
+
+source:
+  url: https://github.com/PaNOSC-ViNYL/libpyvinyl/archive/v{{ version }}.tar.gz
+  sha256: 1e83f89b3b702d6b527bba6a2efd64b4c76cad23ad8a8c305ad356381bcd4e93
+
+  #url: https://pypi.io/packages/source/l/libpyvinyl/libpyvinyl-{{ version }}.tar.gz
+  ##NB: get sha256 from https://pypi.org/project/libpyvinyl/#files
+  #sha256: 5031eacda47519734adf0cea21c684bb368659f08f8df48a698b56fb14c837e3
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+  run:
+    - python >=3.8
+    - pint
+    - dill <=0.3.5.1
+    - scipy
+    - jsons
+    - h5py
+    - json_tricks
+
+test:
+  imports:
+    - libpyvinyl
+  commands:
+    - pip check
+    - pytest .  # [unix]
```

This is noarch, you don't need this selector. However, if there is a significant failure on Windows we need to know so we don't ship a broken package there. Noarch packages are built on linux and ship to all platforms. 

```suggestion
    - pytest .
```

---

Yes, this is just to make the windows builds pass since a few of the pytests were giving false positives there (essentially the upstream project needs to fix their unit tests to also run on windows, a few tests were using wrong path separators). But the package does work on windows.